src/db/client.js

```diff
@@ -107,6 +109,15 @@ async function listTables(server, database) {
   return database.connection.listTables();
 }
 
+async function listViews(server, database) {
+  checkIsConnected(server, database);
+  return database.connection.listViews();
+}
+
+async function listRoutines(server, database) {
```

Comment:
You're right. Also I guess, **async** should be omitted in similar cases with listTables and listDatabases functions?